### PR TITLE
Typo Fixes(SandboxHierarchy.md)

### PR DESCRIPTION
**Pull request by Luro223**

### CODE REVIEW

File: Content/SandboxHierarchy.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 ### Overview
 
-When sandboxed programs create (or modify) objects, such as files, some object must in fact be created. Sandboxie creates these objects out of the way, to protect the system from harmful changes. But these objects must reside somewhere in the system. This page describes where various types of sandboxed objects are placed.
+When sandboxed programs create (or modify) objects, such as files, in fact, some kind of project should be created. Sandboxie creates these objects out of the way, to protect the system from harmful changes. But these objects must reside somewhere in the system. This page describes where various types of sandboxed objects are placed.
```
Comment (isaak654) [inline Content/SandboxHierarchy.md]:
```suggestion
When sandboxed programs create (or modify) objects, such as files, in fact, some kind of data should be created. Sandboxie creates these objects out of the way, to protect the system from harmful changes. But these objects must reside somewhere in the system. This page describes where various types of sandboxed objects are placed.
```

File: Content/SandboxHierarchy.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 
-When sandboxed programs create (or modify) objects, such as files, some object must in fact be created. Sandboxie creates these objects out of the way, to protect the system from harmful changes. But these objects must reside somewhere in the system. This page describes where various types of sandboxed objects are placed.
+When sandboxed programs create (or modify) objects, such as files, in fact, some kind of project should be created. Sandboxie creates these objects out of the way, to protect the system from harmful changes. But these objects must reside somewhere in the system. This page describes where various types of sandboxed objects are placed.
 
 Beginning with version 2.80 of Sandboxie, the layout of the sandbox is not tied to computer-specific device names and account names. See [Portable Sandbox](PortableSandbox.md) for more information.
 
 ### Files
 
-Files are created in the sandbox folder. The hierarchy is as follows:
+Files are created in the sandbox folder in the following hierarchy:
```
Comment (isaak654) [inline Content/SandboxHierarchy.md]:
```suggestion
Files are created in the Sandbox folder according to the following hierarchy:
```

Comment (isaak654) [reply Content/SandboxHierarchy.md]: 2) I would also suggest to replace the remaining `sandbox folder` references in the page with `Sandbox folder`

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by Luro223** [comment]
@isaak654 github shows "isaak654requested changes 2 days ago" even though I accepted all changes :/

**Comment by isaak654** [comment]
It's because GitHub tends to present everything for history purposes. 
With regard to the pull requests that are still open, I'm taking some time for other things.

**Comment by Luro223** [comment]
Okay, got it.

**Review by isaak654 — APPROVED** [review]

**Pull request merged**